jwst/coron/imageregistration.py

```diff
@@ -6,16 +6,17 @@
 from stdatamodels.jwst.datamodels import CubeModel
 
 import logging
+
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
 
-def align_fourierLSQ(reference, target, mask=None):
-    '''LSQ optimization with Fourier shift alignment
+def align_fourier_lsq(reference, target, mask=None):
+    """
+    LSQ optimization with Fourier shift alignment.
 
     Parameters
     ----------
-
         reference : numpy.ndarray
```

Comment:
Indentation should be fixed also while you are at it. Same comment applies to all similar occurrences.